.gitignore

```diff
@@ -3,3 +3,4 @@ tmp/
 internal/devnet/db.bolt
 internal/devnet/log.jsonl
 /chainlink
+.idea/
```

Comment:
This seems specific to your dev environment. Could you move it to your global gitignore?